txpool/txpool.go

```diff
@@ -160,6 +175,8 @@ func (t *TxPool) AddTx(tx *types.Transaction) error {
 	return nil
 }
 
+// addImpl validates the tx and adds it to the appropriate account pool.
+// Additionally, it updates the global valid transactions list
```

Comment:
usage of `pool` and `list` in this sense introduces more confusion, imho

Reply:
Agreed. Renamed 👍 
1f614a8